cob_twist_controller/src/limiters/limiter.cpp

```diff
@@ -223,12 +223,12 @@ KDL::JntArray LimiterAllJointVelocities::enforceLimits(const KDL::JntArray& q_do
             //ROS_WARN("Joint %d Normalized %f", i, q_dot_norm(i));
         }
 
-        if(this->tc_params_.base_active)
-        {
-            q_dot_norm(this->tc_params_.dof) = q_dot_ik(this->tc_params_.dof) / max_factor;
-            q_dot_norm(this->tc_params_.dof + 1) = q_dot_ik(this->tc_params_.dof + 1) / max_factor;
-            q_dot_norm(this->tc_params_.dof + 2) = q_dot_ik(this->tc_params_.dof + 2) / max_factor;
-        }
+        //if(this->tc_params_.base_active)
+        //{
+            //q_dot_norm(this->tc_params_.dof) = q_dot_ik(this->tc_params_.dof) / max_factor;
+            //q_dot_norm(this->tc_params_.dof + 1) = q_dot_ik(this->tc_params_.dof + 1) / max_factor;
+            //q_dot_norm(this->tc_params_.dof + 2) = q_dot_ik(this->tc_params_.dof + 2) / max_factor;
+        //}
```

Comment:
apply limiters in inverse_differential_kinematics_solver rather than twist_controller....then this can be added back in...